packages/create-atomic/templates/scripts/clean-up.js

```diff
@@ -0,0 +1,19 @@
+const {unlinkSync, readFileSync, writeFileSync} = require('fs');
+const {join} = require('path');
+const {getPackageManager} = require('./utils');
+const pkgJson = JSON.parse(readFileSync('package.json'));
+
+['postinstall', 'setup-env', 'setup-server', 'setup-cleanup'].forEach(
```

Comment:
Good idea. I think we should do the same for the other frameworks.

Reply:
Wasn't my idea 😉